project_risk/README.rst

```diff
@@ -0,0 +1,102 @@
+============
+Project Risk
+============
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fproject-lightgray.png?logo=github
+    :target: https://github.com/OCA/project/tree/11.0/project_risk
+    :alt: OCA/project
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/project-11-0/project-11-0-project_risk
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/140/11.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+With this module you can manage your projects risk using the MOR method.
+
+https://www.axelos.com/best-practice-solutions/mor/what-is-mor
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+This module provides a number of default risk categories, but you can add more at *Project > Configuration > Risk Categories*.
+For risk response categories you need to go to *Project > Configuration > Risk Response Categories*.
+
+Usage
+=====
+
+To define risks for your projects:
+
+#. Go to *Project > Search > Risks*;
+#. create a new risk;
+#. the rating is automatically calculated based on this risk matrix:
+    .. image:: https://raw.githubusercontent.com/OCA/project/11.0/project_risk/static/description/matrix.png
+       :alt: Risk matrix
+#. set the state to 'Active' if the risk is in effect;
+#. set the state to 'Closed' if the risk is processed or e.g. completely avoided.
+
+The list of risks is by default ordered by rating and proximity.
+Risks with a rating higher than rating 'Medium-High' become red.
+
+Known issues / Roadmap
+======================
+
+* Create task by risk response
```

Comment:
Possible detail about the issue creating task  ?

Reply:
Hi, I made the roadmap more clear. Can you re-review, please?